src/components/Template.tsx

```diff
@@ -74,13 +74,12 @@ export const Template = () => {
                   : ""
               }
             >
-              <button
-                type="button"
-                onClick={forceSave}
-                className="dp-button button-medium primary-green"
-              >
-                <FormattedMessage id="save" />
-              </button>
+              <>
+                {
+                  // Empty item added because the last item is on the right
+                  // TODO: restructure header styles and ed-header-list class
+                }
+              </>
```

Comment:
@GusBaamonde this is the patch we talked about. What do you think about applying it as a temporary solution?

Reply:
Refactor: https://github.com/FromDoppler/doppler-editors-webapp/pull/442#discussion_r1122243695